packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -119,7 +124,7 @@ export type Props = {
      * on the state of the interactive graph previewed at the bottom of the
      * editor page.
      */
-    correct: any; // TODO(jeremy)
+    correct: PerseusGraphType;
```

Comment:
Not sure if this is the right time to think about this, but do we want `correct` to be typed as `PerseusGraphType` long term?

Personally, I think it makes more sense for `correct` to only have the `type` and `coords` fields in it, and all the other field would be specific to `graph`. And in most of the places where `correct` is being used now, we would use `graph` instead. We could start with `PerseusGraphType` for now and make that change later?

Any thoughts on that? (@benchristel @jeremywiebe)

I could see this potentially turning into a larger discussion, so I think it could also be worth taking this offline.

Reply:
I agree that would be better. I think it's a bigger effort though (I'd like to analyze our existing content to figure out what values for `correct` are in use). I'll add a TODO comment and create a ticket.